quiz_app/questions/static/isCorrect.js

```diff
@@ -0,0 +1,57 @@
+const btns = document.querySelectorAll("button");
```

Comment:
Using js OK, but I suggest to use Django forms for this, just for practice sake.
It should handle validation etc. as well.